### DISCUSSION TITLE
CKTON Liquidity Boost: Treasury Proposal

### DISCUSSION THREAD

**Discussion by Laki389** [darwinia-network/darwinia#1500] [General]
1. **Brief introduction, issue explanation and solution.**

1.1 **Brief introduction.**
Name: CKTON (Crab Commitment Token)
Total Supply: 220 116

1.2 **Issue explanation.**
CKTON is not traded on any CEX. :worried:
It is traded on only one DEX, https://snowswap.xyz/ with an extremely insignificant liquidity of only 8 CKTON.
Past experience shows that the community does not provide liquidity because the staking rewards are higher and the risk is lower.

1.3 **Solution.** :white_check_mark:
**I propose an incentive of 1 000 000 (one million) $crab, for CKTON - CRAB lp farm at https://snowswap.xyz/, for a duration of 180 days.**

That's about 5555 $crab incentive per day, which I expect to bring at least 4500 ckton liquidity after a few weeks from the start of the incentive.
4500 ckton is currently about 2% of Total Supply.

For comparison, currently 1 ckton staking gives 0.585 crab/day, noting that much of the ckton that was previously staked have not been migrated to the new staking website https://kton-staking.darwinia.network/.
Taking that into account, it is to be expected that in the near future staking rewards will be 0.4 crab/day or less.
This is where my optimism is based that the target of 4500 ckton will be reached.

The sum of 1 million $crab will be provided by the Crab Treasury.

Go ahead with your thoughts and feedback. :+1:

**Comment by SasoLithops** [comment]
I support this proposal as my Ckton is stuck in staking alone and impossible to get more with Crab rewards due to current poor liquidity.

**Comment by 2xJDubs** [comment]
Think this is a great idea and we have seen this work well with the KTON/RING pool on Darwinia.  To help support this proposal there is plenty of CRAB in the treasury.  Currently the CRAB Treasury holds 1.093B CRAB so this represents less than 0.1% of what the treasury holds.

We do need the team to weigh in on if this impacts long term staking rewards as there is no longer inflation on CRAB and all staking rewards come from the treasury.  @wuminzhe can you please help with this?

**Comment by JuneCribb** [comment]
Good proposal @Laki389 
Although I am not a holder of CKTON, I'm fully in support. 👏

**Comment by hackfisher** [comment]
To proceed,

1. For gov tool of Crab, can use https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fcrab-rpc.darwiniacommunitydao.xyz#/referenda, please seek help in discord governance channel from @AurevoirXavier if you have any question about how to submit proposal or question about usage.

Refer: https://docs.darwinia.network/community/guide/governance/

2. For snowswap farm cooperation or farm contract address and creation, seeking help from snowswap or @WoeOm

**Comment by johneggleston** [comment]
Sounds cool, I think the most opportunity that this presents is mainly to bring Crab out of obscurity a bit, I’m sure that more than 90% of Darwinia investors wouldn’t have any idea what it’s purpose is. Or even if it’s a good investment compared to RING and KTON.
